api/views.py

```diff
@@ -737,8 +736,8 @@ class StructureAssignGenericNumbers(views.APIView):
 
     def post(self, request):
 
-        # root, ext = os.path.splitext(request._request.FILES['pdb_file'].name)
-        generic_numbering = GenericNumbering(StringIO(request._request.FILES['pdb_file'].file.read().decode('UTF-8',"ignore")))
+        root, ext = os.path.splitext(request.FILES['pdb_file'].name)
+        generic_numbering = GenericNumbering(StringIO(request.FILES['pdb_file'].file.read().decode('UTF-8',"ignore")))
```

Comment:
Why is request._request not necessary anymore?
I added this recently to fix the functioning of these calls on the server with PR #172 .